control/config.go

```diff
@@ -152,71 +152,6 @@ func GetDefaultConfig() *Config {
 	}
 }
 
-// UnmarshalJSON unmarshals valid json into a Config.  An example Config can be found
```

Comment:
[`json.Unmarshal`](https://github.com/intelsdi-x/snap/blob/master/pkg/cfgfile/cfgfile.go#L120-L121) decodes the json with the default algorithm, unless `UnmarshalJSON` is defined for a type. In this particular case , `control.Config.UnmarshalJSON` implements the default algorithm. The only value would be if the JSON is not valid the logs give more information. But this case is already catch by [the schema validation](https://github.com/intelsdi-x/snap/blob/master/pkg/cfgfile/cfgfile.go#L115-L119). So it sounds more like code duplicate.